recipes/metawrap/meta.yaml

```diff
@@ -0,0 +1,81 @@
+{% set name = "metawrap" %}
+{% set version = "1.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/bxlab/metaWRAP/archive/v{{ version }}.tar.gz
+  md5: face9197bb696b037d7b2acdf27c668a
+
+build:
+  number: 0
+  skip: True # [osx]
+
+requirements:
+  build: 
+
+  run:
+    - bcftools 1.6
+    - biopython 1.69
+    - blast 2.7.*
+    - bmfilter 3.101
+    - bmtagger 3.101
+    - bmtool 3.101
+    - boost 1.64.0
+    - boost-cpp 1.64.0
+    - bowtie2 2.3.*
+    - brewer2mpl 1.4.1
+    - bwa 0.7.15
+    - cairo 1.14.8
+    - checkm-genome 1.*
+    - concoct 0.4.0
+    - fastqc 0.11.5
+    - gnutls 3.5.15
+    - icu 58.2
```

Comment:
From where did you get this list? Is this an environment file originally.

Reply:
No, this is not an environment file. MetaWRAP relies on dozens of software to work, as it workflow incorporates many aspects of metagenomic analysis.

Reply:
I should also add that this list has been curated and worked over dozens of iterations and is well refined at this point. See the MetaWRAP conda distribution: https://github.com/bxlab/metaWRAP/tree/master/conda_pkg and https://anaconda.org/ursky/metawrap-mg/files.